Makefile

```diff
@@ -14,3 +15,11 @@ sunxi-%.o: sunxi-%.c
 
 clean:
 	rm -f *.o sunxi-fw
+
+install: sunxi-fw
+	install -Dm755 sunxi-fw $(PREFIX)/bin/sunxi-fw
```

Comment:
Can you please split the "D" and "m755" up (-D -m755), it looks a bit weird to combine switches and options with arguments. Also please add "-s", to strip the binary before installation.

Reply:
fixed